linkerd/examples/zipkin.yaml

```diff
@@ -10,6 +10,8 @@ routers:
   servers:
   - port: 4140
 
-tracers:
+telemetry:
 - kind: io.l5d.zipkin
```

Comment:
should probably add the io.l5d.commonMetrics telemeter here as well so that the admin dashboard works